Lib/profiling/tracing/__init__.py

```diff
@@ -0,0 +1,207 @@
+"""Tracing profiler for Python.
+
+This module provides deterministic profiling of Python programs by tracing
+every function call and return. It's based on the cProfile implementation.
```

Comment:
In the future we will have removed cProfile, so this sentence won't make sense. Perhaps more relevant for the docs?
```suggestion
every function call and return.
```